python/lsst/ts/nightreport/nightreport.py

```diff
@@ -0,0 +1,77 @@
+__all__ = [
+    "Telescope",
+    "NightReport",
+    "NIGHTREPORT_FIELDS",
+    "NIGHTREPORT_ORDER_BY_VALUES",
+]
+
+import datetime
+import enum
+import uuid
+
+from pydantic import BaseModel, Field
+
+
+class Telescope(str, enum.Enum):
+    auxtel = "AuxTel"
+    maintel = "Simonyi"
+
+
+class NightReport(BaseModel):
+    id: uuid.UUID = Field(title="Night report ID: a UUID that is the primary key.")
```

Comment:
Please add a class doc string here and everywhere else.

Is there a reason why this class only has class variables and no instance variables? If there is please add that to the class doc string.

Reply:
This is the way of Pydantic model definition, I added some reference on the docstrings: https://docs.pydantic.dev/latest/.